docs/releases/3.1.txt

```diff
@@ -27,6 +27,27 @@ officially support the latest release of each series.
 What's new in Django 3.1
 ========================
 
+Automatic :class:`~django.apps.AppConfig` detection
```

Comment:
`s/detection/discovery/`? 

We have "discovery" elsewhere. I think one or the other (always) helps folks know we're (still) talking about the same thing.